keps/sig-api-machinery/3962-mutating-admission-policies/README.md

```diff
@@ -1032,8 +1032,8 @@ We will test the edge cases mostly in integration test and unit test. We may add
 
 #### Beta
 
-- Gather feedback from developers and surveys
-- Complete features A, B, C
+- Add metrics for MAP admission plugin
+- Fix issues/bugs been brought up
```

Comment:
Do we have any known ones?  If so let's list them them here.  Otherwise just state "Fix any blocking issues/bugs surfaced before 1.34 code freeze"

Reply:
Updated. Thanks!